zjit/src/backend/lir.rs

```diff
@@ -1397,7 +1397,7 @@ impl Assembler
                 Some(Opnd::VReg { idx, .. }) => Some(*idx),
```

Comment:
> // If the output VReg of this instruction is used by another instruction,

This comment needs some modification too. We now allocate a register for VReg not used by another instruction.